README.md

```diff
@@ -45,8 +44,8 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- GitHub: [@githubhandle](https://github.com/JulianaOsemeke)
+- Twitter: [@twitterhandle](@JulianaOsemeke)
```

Comment:
- [ ] Replace the placeholder social accounts with your actual social usernames.
```suggestion
- GitHub: [@JulianaOsemeke](https://github.com/JulianaOsemeke)
- Twitter: [@JulianaOsemeke](https://twitter.com/JulianaOsemeke)
```